scripts/browserifyTestDependency.js

```diff
@@ -0,0 +1,66 @@
+/*!
+Copyright 2016 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+The research leading to these results has received funding from the European Union's
+Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+// The creation of this script is to work around the issue of having to run a npm postinstall step
+// only within the context of a GPII development environment.
+// This script uses fluid-resolve npm package (https://www.npmjs.com/package/fluid-resolve) to detect
+// the existence of %universal/node_modules/browserify directory that is only pulled in as a devDependency.
```

Comment:
Under npm 3 this will not in fact be the path of browserify - just say "the existence of a resolvable browserify module which is ordinarily only pulled in as a devDependency"